forest/src/cli/mod.rs

```diff
@@ -128,6 +133,9 @@ impl DaemonOpts {
 
         cfg.network.kademlia = self.kademlia.unwrap_or(cfg.network.kademlia);
         cfg.network.mdns = self.mdns.unwrap_or(cfg.network.mdns);
+        if let Some(target_peer_count) = &self.target_peer_count {
+            cfg.network.target_peer_count = *target_peer_count;
+        }
```

Comment:
You don't need to reference and dereference here.

Reply:
done